### PR TITLE
Steams dev/issue16

### PR DESCRIPTION
**Pull request by SteamsDev**
This is what I have for Save/Load. There might be conflicts so I will need help resolving them.

### CODE REVIEW

File: gameapplication.py [hunk 1]
```diff
@@ -14,6 +15,21 @@
 puzzleStarted = 0
 gaUserLetters = "empty"
 gaReqLetter = "empty"
+getList = list()
+
+def gameLoad():
```
Comment (tmhughes1) [inline gameapplication.py:20]: Please add comments so silly me can know what's going on!

File: loadgame.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+def loadUserLetters(save):
```
Comment (tmhughes1) [inline loadgame.py:1]: Looks like you have all the necessary components here! I wonder if we should also save the puzzle ie. how it looked/the order of the letters being displayed.

Comment (tmhughes1) [reply loadgame.py:1]: I guess I don't know if this is already done which is why comments would be very helpful x3

**Review by tmhughes1 — CHANGES_REQUESTED** [review]
Please add comments so everyone can understand the code!

**Comment by tmhughes1** [comment]
Also get rid of your pychache folder!

File: loadgame.py (hunk 2 above)
Comment (SteamsDev) [reply loadgame.py:4]: Comments have been added. Let me know if there's anything specific you still need.

File: gameapplication.py (hunk 1 above)
Comment (SteamsDev) [reply gameapplication.py:21]: Added.

**Review by tmhughes1 — APPROVED** [review]

**Pull request merged**